simSPI/linear_simulator/test_data_creator.py

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
should this be in tests?

Reply:
Removed this file. Actually, it was just a script to create the data for tests. Not needed in the repo.